db/seeds.rb

```diff
@@ -1,8 +1,15 @@
 EMAIL = 'test@example.com'
 PASSWORD = 'password'
+ADMIN_EMAIL = 'test@example.com'
```

Comment:
管理者画面はデフォルトのメールアドレスにしていただけますか。

```suggestion
ADMIN_EMAIL = 'admin@example.com'
```